app/services/project_data_service.rb

```diff
@@ -0,0 +1,90 @@
+require "base64"
+
+class ProjectDataService
+
+  def initialize(project, access_token)
+    @project = project
+    @access_token = access_token
+    @client = Octokit::Client.new(access_token: access_token)
+    @circle_config = circle_config
+  end
+
+  def github_org
+    Organization.find(@project.organization_id).name
+  end
+
+  def project_name
+    url = git_remote.split("/").last
+    url.gsub(".git", "")
+  end
+
+  def git_remote
+    stage = @project.stages&.detect{ |s| s.name == "master" }
+    stage && stage.git_remote
+  end
+
+  def computed_properties
+    {
+      base_libraries: {
+        ruby: ruby_version,
+        node: node_version
+      },
+      ci_provider: ci_provider,
+      renovate: has_renovate?,
+      orbs: orbs
+    }
+  end
+
+  def ci_provider
+    @circle_config && "circleci"
```

Comment:
```suggestion
    'circleci' if @circle_config
```

(Equivalent but slightly more ruby-ish.)